crates/pipeline_manager/src/bin/prober-server.rs

```diff
@@ -0,0 +1,58 @@
+use std::sync::Arc;
```

Comment:
Better named probe-server.

Reply:
The main motivation for prober-server is for consistency to compiler-server -- there's also the naming conflict with the database probe type in terms of the package inside pipeline-manager.